Dockerfile

```
@@ -15,6 +15,10 @@ RUN apk update && apk upgrade && \
 
 WORKDIR /usr/src/app
 RUN mkdir data
+
+RUN addgroup -g 190 -S infracost && adduser -u 190 -S infracost -G infracost
+RUN ["chown", "-R", "infracost:infracost", "/usr/src/app"]
```

I would use uid 1001 since that seems to be convention since uids < 1000 are reserved for system processes. We could compress this to a single `RUN` as well so we don't create a new layer.

```suggestion
RUN addgroup -g 1001 -S infracost && \
    adduser -u 1001 -S infracost -G infracost && \
    chown -R infracost:infracost /usr/src/app
```